Minimal_API/Program.cs

```diff
@@ -0,0 +1,57 @@
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseHttpsRedirection();
+
+app.MapPost("/calculator", (Calculator calc) =>
+{
+    switch (calc.OP)
+    {
+        case "+":
+            return Results.Json(calc.N1 + calc.N2);
+        case "-":
+            return Results.Json(calc.N1 - calc.N2);
+        case "*":
+            return Results.Json(calc.N1 * calc.N2);
+        case "/":
+            return Results.Json(calc.N1 / calc.N2);
+    }
+    return Results.Json("Opration not supported");
+});
+
+
+app.Run();
+
+class Calculator
```

Comment:
Move to another file

Reply:
Moved the Calculator class to another file.